go/pkg/sysdb/metastore/db/dao/collection.go

```diff
@@ -30,7 +30,7 @@ func (s *collectionDb) DeleteAll() error {
 func (s *collectionDb) GetCollectionEntry(collectionID *string, databaseName *string) (*dbmodel.Collection, error) {
 	var collections []*dbmodel.Collection
 	query := s.db.Table("collections").
-		Select("collections.id, collections.name, collections.database_id, collections.is_deleted, databases.tenant_id, collections.version, collections.version_file_name, collections.root_collection_id").
+		Select("collections.id, collections.name, collections.database_id, collections.is_deleted, collections.tenant, collections.version, collections.version_file_name, collections.root_collection_id").
```

Comment:
my understanding is that this annotation controls what `dbmodel.Collection.Tenant` pulls from:
https://github.com/chroma-core/chroma/blob/a23754123e79fe1ef702c7b2237f0b1f4be9dbbc/go/pkg/sysdb/metastore/db/dbmodel/collection.go#L29